### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by jdespatis**
It seems title and body cannot be null (I've an error thrown if null)
and `setRouteContent` gives me an error, `setContent` is ok

### CODE REVIEW

File: book/routing.rst [hunk 1]
```diff
@@ -478,8 +478,10 @@ follows::
             $content = new StaticContent();
             $content->setParentDocument($dm->find(null, '/cms/content'));
             $content->setName('my-content');
+            $content->setTitle('My Content');
+            $content->setBody('Some Content');
```
Comment (dbu) [inline book/routing.rst]: ups, you are right about this. i think we cleaned that up a bit later, after we wrote this doc.

sorry for that and thanks for fixing it

File: book/routing.rst [hunk 2]
```diff
@@ -505,8 +507,8 @@ also ``/projects`` as there is a default for the id parameter.
     doing it in a fixture like this).
 
 Because you defined the ``{id}`` route parameter, your controller can expect an
-``$id`` parameter. Additionally, because you called ``setRouteContent`` on the
-route, your controller can expect the ``$contentDocument`` parameter.
+``$id`` parameter. Additionally, because you called ``setContent`` on the
+route, your controller can expect the ``$content`` parameter.
```
Comment (dbu) [inline book/routing.rst]: setRouteContent => setContent is correct as well. but the request attribute is called contentDocument. that comes from DynamicRouter::CONTENT_KEY

i will fetch this commit into the 1.1 branch of the doc (master is for the next upcoming version) and adjust this line.

thanks @jdespatis

**Comment by dbu** [comment]
thanks a lot, pushed to 1.2 in 684d670888dc661c508dc33a93d720339ff85c32

File: book/routing.rst (hunk 2 above)
Comment (jdespatis) [inline book/routing.rst]: ok @dbu thanks

Comment (dbu) [inline book/routing.rst]: np, thanks for the fix. keep them coming if you encounter other issues.

Comment (jdespatis) [inline book/routing.rst]: For sure @dbu !
Hope I'll have time to continue the reading of the doc

Btw, it seems impossible to download the pdf for :
* Symfony CMF Bundles
* Symfony CMF Components
* Symfony CMF Cookbook

=> Would it be possible to generate them as the Symfony CMF book is already done?
Much easier to read a pdf in offline mode indeed

Comment (dbu) [inline book/routing.rst]: @jdespatis best open an issue asking about this on symfony-docs - the doc is integrated into the symfony doc and the problems must be on their side (or they need to explain to us how we can fix it)

Comment (jdespatis) [inline book/routing.rst]: @dbu you're right, let's go for it: https://github.com/symfony/symfony-docs/issues/5048

**Pull request closed without merging**